apps/web/src/components/EditorHeader/index.tsx

```diff
@@ -234,6 +241,55 @@ export default function EditorHeader({
           onChange={onModelChange}
         />
       </FormFieldGroup>
+      {selectedProvider === envClient.NEXT_PUBLIC_DEFAULT_PROJECT_ID && (
+        <div>
+          {freeRunsCount !== undefined ? (
+            <Text.H6 color='foregroundMuted'>
+              You have consumed{' '}
+              <Tooltip
+                trigger={
+                  <strong className='text-primary'>
+                    {freeRunsCount} of 100 daily free runs.
```

Comment:
I think there's a global constant for the limit? Use it here too.

Reply:
the constant and the number are different (constant is 1k, but we wanna say 100 here)

Reply:
that constant and this number are different

Reply:
What? Why? Isn't it the daily limit for free runs?

Reply:
talked offline